packages/dd-trace/src/profiling/exporters/console.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+/* eslint-disable no-console */
+
+const { inspect } = require('util')
+
+class ConsoleExporter {
+  constructor (options = {}) {
+    this._json = !!options.json
+  }
+
+  async export ({ profiles }) {
```

Comment:
This method does not need to be async.

Reply:
The caller expects a promise. What would you recommend instead?

Reply:
It doesn't matter unless you're specifically calling `.then()`, because `await` works on non-promises as well.

Reply:
Similar to the above, this behavior cannot be relied on since there is no explicit async-specific construct.

Reply:
If the insistence is on the caller always receiving a promise, then that needs to be documented, because otherwise this looks like an incorrect usage of async.

Reply:
This file is gone, so we'll call this resolved.